src/routes.py

```diff
@@ -1,17 +1,21 @@
 import os
 import smtplib
 
-from flask import render_template, request, make_response, before_render_template
+from flask import render_template, request
 from flask import current_app as app
-from .Database import db, Users, Stores, Orders, Items
+from .Database import db, Users, Stores
 
 
 @app.route("/")
 def home():
     title = "Fashi.on the way"
+    # Nike store >>>
     store1 = Stores.query.get(1)
```

Comment:
you can change the name of the variable to be more readable and then you don't need the comment.
for example instead of `store1` you can call it `nike_store`. 👈 I prefer this one!

Reply:
Done!